data/console/ansible/hosts

```diff
@@ -0,0 +1,3 @@
+[all]
+localhost
```

Comment:
```suggestion
localhost ansible_connection=ssh
```

Following https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/15684#issuecomment-1281118815 you could try to explicitly set `ansible_connection=ssh` so that ansible uses ssh. In this setup there should not be much of a difference between ansible running against localhost or another machine. It does not cover all use cases (e.g. dependencies on the remote host might still be a problem), but it would for sure cover the network dependent ansible connection.

Just a suggestion.